tests/PHPUnit/Integration/EmailValidatorTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Tests\Integration;
+
+use Piwik\Piwik;
+use Piwik\Tests\Framework\TestCase\IntegrationTestCase;
+
+/**
+ * @group Core
+ */
+class EmailValidatorTest extends IntegrationTestCase
+{
+    protected function isValid($email)
+    {
+        return Piwik::isValidEmailString($email);
+    }
+
+    public function testValidStandard()
```

Comment:
The current convention for test names is `test_{{methodName}}_{{descriptionOfTest}}`, for example:

`test_isValid_shouldValidateANormalEmail`

Can you change the test method names to this type of name?

More examples: https://github.com/piwik/piwik/blob/master/tests/PHPUnit/Integration/TrackerTest.php